src/navigation/NavbarTwoColumns.tsx

```diff
@@ -10,7 +10,7 @@ type INavbarProps = {
 const NavbarTwoColumns = (props: INavbarProps) => (
   <div className="flex flex-wrap justify-between items-center">
     <div>
-      <Link href="/">
+      <Link href="/" legacyBehavior>
```

Comment:
The PR looks good to me, just one question about the `legacyBehavior`. it's not possible to use the modern `<Link>` component?